worker/worker/worker.js

```diff
@@ -230,6 +230,7 @@ async function handler_witness_instagram_post(request) {
 
     let pk = decodeURI(searchParams.get("pk"));
     let handle = searchParams.get("handle");
+    let sig_type = searchParams.get("sig_type");
```

Comment:
Maybe put a default here in case we have mismatched versions that are deployed?

Reply:
I realise this is just for Instagram. Feel free to ignore my comment. But maybe those common parameters should be grouped together for all endpoints.